arangopipe/tests/AQL/Running_AQL.md

```diff
@@ -0,0 +1,81 @@
+# Running AQL queries  on Arangopipe
+
+Apart, from using the **Arangopipe** search feature, it is also possible to query machine learning meta-data using the query feature in **ArangoDB**. An excerpt of the feature is shown in the figure below.
+<figure>
+  <img src="AQL_feature.png" alt="my alt text" width=90%/>
+  <figcaption style ="text-align:center">Figure 1.</figcaption>
+</figure>
+The left pane shows an editor area where we can enter queries. We can the execute the query by clicking on the *_Execute_* button in the lower right hand corner. Let us illustrate this with an example. The dataset used for illustration is the wine dataset used to illustrate how **Arangopipe** can be used with *_mlflow_* (see the *_mlflow_* folder). We will illustrate the following operations:
+
+1. Deleting a dataset from the database
```

Comment:
Should we maybe add a graph traversal query and not just DDL?

Reply:
Sure, I just need to add some cases to insert some run information into the database before querying it. Yeah, that makes that complete.